include/veriblock/storage/block_repository_rocks.hpp

```diff
@@ -232,6 +231,63 @@ class BlockRepositoryRocks : public BlockRepository<Block> {
     }
     return result;
   }
+
+ private:
+  class CursorRocks : public Cursor<height_t, stored_block_t> {
+   private:
+    std::vector<rocksdb::Iterator*> rocks_its;
+
+   public:
+    CursorRocks(const BlockRepositoryRocks<stored_block_t>& repo) {
+      repo._db->NewIterators(
+          rocksdb::ReadOptions(),
+          {repo._heightHashesHandle.get(), repo._hashBlockHandle.get()},
+          &rocks_its);
+    }
+
+    void seekToFirst() override {
+      rocks_its[0]->SeekToFirst();
+      rocks_its[1]->SeekToFirst();
+    }
+
+    void seek(const height_t& key) override {
+      rocks_its[0]->Seek(std::to_string(key));
```

Comment:
Height may contain multiple hashes. We removed height related API from the BlockchainRepo interface so it does not matter.

Reply:
fixed